README.md

```diff
@@ -211,7 +211,9 @@ particularly in cloud native environments where everything is software-defined.
 Weekly meetings at 8:00 AM PT (50 min) (see your timezone [here](https://time.is/0800_today_in_PT?CNCF_Security_TAG_Supply_Chain_WG_Meeting))
 See [CNCF calendar](https://www.cncf.io/calendar/) for invite.
```

Comment:
since we are updating this, can we update meeting info to reflect the new changes.